src/babel/index.js

```diff
@@ -744,25 +745,39 @@ function isPlainTemplateLiteral(t, node) {
   return t.isTemplateLiteral(node) && !node.expressions.length && node.quasis.length === 1
 }
 
+function isNumberLiteral(t, node) {
+  return t.isNumericLiteral(node) || (t.isUnaryExpression(node) && node.operator === '-' && node.prefix && t.isNumericLiteral(node.argument))
+}
+
 function validateStyleset(t, styleset) {
   if (!t.isObjectExpression(styleset.node)) {
     throw styleset.buildCodeFrameError(
-      "ZACS StyleSheets must be simple object literals, like so: `{ backgroundColor: 'red', height: 100 }`. Other syntaxes, like `foo ? {xxx} : {yyy}` or `...styles` are not allowed.",
+      "ZACS StyleSheets must be simple object literals, like so: `text: { backgroundColor: 'red', height: 100 }`. Other syntaxes, like `foo ? {xxx} : {yyy}` or `...styles` are not allowed.",
     )
   }
 
   const properties = styleset.get('properties')
   properties.forEach(property => {
-    if (!isPlainObjectProperty(t, property.node)) {
+    if (!isPlainObjectProperty(t, property.node, true)) {
```

Comment:
Does it mean we're supporting prefixed properties like `--webkit-*` as well?

Reply:
yes, `WebkitFoo` or `MozFoo` get compiled into `-webkit-foo`, `-moz-foo` -- same as when configuring CSS in JS using DOM/React apis